ppml/src/main/cpp/quote_verification.cpp

```diff
@@ -86,21 +86,71 @@ int ecdsa_quote_verification(vector<uint8_t> quote) {
+        quote_verification_result);
+    printf("The SGX platform firmware and SW are at the latest security patching level"
+            "but there are platform hardware configurations"
+            "that may expose the enclave to vulnerabilities.\n")
+    ret = 1;
+    break;
   case SGX_QL_QV_RESULT_OUT_OF_DATE:
+    printf(
+          "\tWarning: Verification completed with Non-terminal result: %x\n",
+          quote_verification_result);
+    printf("The SGX platform firmware and SW are not at the latest security patching level."
+            "The platform needs to be patched with firmware and/or software patches.\n")
+      ret = 1;
+      break;
   case SGX_QL_QV_RESULT_OUT_OF_DATE_CONFIG_NEEDED:
+    printf(
+          "\tWarning: Verification completed with Non-terminal result: %x\n",
+          quote_verification_result);
+    printf("The SGX platform firmware and SW are not at the latest security patching level."
+           "The platform needs to be patched with firmware and/or software patches.\n")
+      ret = 1;
+      break;
   case SGX_QL_QV_RESULT_SW_HARDENING_NEEDED:
+    printf(
+              "\tWarning: Verification completed with Non-terminal result: %x\n",
+              quote_verification_result);
+    printf("The SGX platform firmware and SW are at the latest security patching level"
+     "but there are certain vulnerabilities that can only be mitigated with"
+      "software mitigations implemented by the enclave.\n")
+       ret = 1;
+       break;
   case SGX_QL_QV_RESULT_CONFIG_AND_SW_HARDENING_NEEDED:
     printf(
         "\tWarning: Verification completed with Non-terminal result: %x\n",
         quote_verification_result);
+    printf("The SGX platform firmware and SW are at the latest security patching level"
+            "but there are certain vulnerabilities that can only be mitigated with"
+            "software mitigations implemented by the enclave.\n")
     ret = 1;
     break;
   case SGX_QL_QV_RESULT_INVALID_SIGNATURE:
+    printf("\tError: Verification completed with Terminal result: %x\n",
+             quote_verification_result);
+    printf("\tThe signature over the application report is invalid\n")
+      ret = -1;
+      break;
   case SGX_QL_QV_RESULT_REVOKED:
+    printf("\tError: Verification completed with Terminal result: %x\n",
+             quote_verification_result);
+    printf("\tThe attestation key or platform has been revoked\n")
+      ret = -1;
+      break;
   case SGX_QL_QV_RESULT_UNSPECIFIED:
+    printf("\tError: Verification completed with Terminal result: %x\n",
+             quote_verification_result);
+    printf("\tThe Quote verification failed due to an error in one of the input\n")
+      ret = -1;
+      break;
   default:
     printf("\tError: Verification completed with Terminal result: %x\n",
            quote_verification_result);
+    printf("please refer to https://download.01.org/intel-sgx/latest/dcap-latest/linux"
```

Comment:
This print is only called when it equal ot `SGX_QL_QV_RESULT_MAX`, due to breaks in the above conditions.

Move this print code out of switch. Then, print it out when ret == -1.